releaseNewVersion.sh

```diff
@@ -70,8 +70,10 @@ select yn in "major-new-release" "feature-release" "minor-fix"; do
     esac
 done
 echo -e "\n"
-# now replace the version in packages.json and version.js (inplace!)
-perl -pi -e "s/\"version\": \"${packagesVersion}\"/\"version\": \"${newVersion}\"/" package.json
+# now replace the version in NPM files and version.js (inplace!)
+npm version "${newVersion}"
```

Comment:
>    If run in a git repo, it will also create  a  version  commit  and  tag.
       This  behavior  is controlled by git-tag-version (see below), and can be
       disabled on the command line by running  npm  --no-git-tag-version  ver-
       sion.  It will fail if the working directory is not clean, unless the -f
       or --force flag is set.
       
So I think we want to run with `--no-git-tag-version`, as we use our own tagging from the release branch. (There's really no need for a release branch, we should just tag the master branch and publish from that... but that's beyond this PR!)

Reply:
I overlooked this option! Updated to use `--no-git-tag-version` in force push.